worlds/hk/Items.py

```diff
@@ -61,6 +61,7 @@ class HKItemData(NamedTuple):
     "VesselFragments": lookup_type_to_names["Vessel"],
     "WhisperingRoots": lookup_type_to_names["Root"],
     "WhiteFragments": {"Queen_Fragment", "King_Fragment", "Void_Heart"},
+    "ProgressiveDreamNail": {"Dream_Nail", "Dream_Gate", "Awoken_Dream_Nail"},
```

Comment:
I dislike that this is inconsistent with the rest of the group names, i think it would be better named similarly to claw and friends (idk to what) or else have all the other manually made item groups renamed to progressive whatever as well. Also may as well use spaces in the name, people will be typing them

Reply:
Seconding this, I think "DreamNails" could be better to match the other groups

Reply:
there will be no spaces in the other group names (because they are the RandomizeX option pools with the Randomize prefix stripped) so i see no reason to break from that for this
I also don't have a better short name, the reason I chose ProgressiveDreamNail is because that is likely what the item name that replaces the concept of this item group will be, so habits don't have to change when the underlying code does
as for renaming all the claw and friends to ProgressiveX that's probably fine but a bigger change than i wanted this PR to be

Reply:
to be clear, i didn't see medic's suggestion, `DreamNails` could be ok,
my gut is worried it's too close to `Dream_Nail`, but no strong feelings rn

Reply:
I think DreamNails is fine though it would be nice to test and have an assurance it won't get fuzzy matched in an unfortunate way.

Regarding spaces I meant that if people are looking for "progressive dream nail" they will put the spaces in it like every other game has (and it would be the eventual item name)

Reply:
tested DreamNails and 
`Dream Nail` finds the item
`DreamNails` finds the group
`DreamNail` reports back `Too many close matches for 'dreamnail', did you mean 'DreamNails'? (90% sure)`

seems fine to me